### PR TITLE
[TEVA-2394] Remove Account Creation CTA A:B Test and Related Code

### PR DESCRIPTION
**Pull request by josephhull676**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-2394

### Screenshot of CTA

Screenshot 2021-04-12 at 11 19 48

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-3250.london.cloudapps.digital>

File: app/views/home/_signed_in_jobseeker.html.slim [hunk 1]
```diff
@@ -2,7 +2,4 @@ h2.govuk-heading-m = t(".title")
 
 p.govuk-body = t(".description")
 
-- if current_variant?(:"2021_03_jobseeker_account_cta_test", :colour)
-  = govuk_link_to t("buttons.view_account"), jobseeker_root_path, class: "govuk-!-margin-bottom-0", button: true
-- else
-  = govuk_link_to t("buttons.view_account"), jobseeker_root_path, class: "govuk-button--secondary govuk-!-margin-bottom-0", button: true
+= govuk_link_to t("buttons.view_account"), jobseekers_saved_jobs_path, class: "govuk-button--secondary govuk-!-margin-bottom-0", button: true
```
Comment (david-mears-dfe) [inline app/views/home/_signed_in_jobseeker.html.slim:5]: Can we keep using `jobseeker_root_path`?

In routes.rb:
```ruby
get "/jobseekers/saved_jobs", to: "jobseekers/saved_jobs#index", as: :jobseeker_root
```

**Comment by alexbowen** [comment]
can you remove the visual test snapshots for this ab test also?

**Review by alexbowen — APPROVED** [review]
small thing i just messaged you about else 👍

**Review by david-mears-dfe — APPROVED** [review]

**Pull request merged**